database/models/Contact.js

```diff
@@ -0,0 +1,11 @@
+const bookshelf = require('./bookshelf');
+
+class Contact extends bookshelf.Model {
+  get tableName() {return 'createdBy';}
+  get hasTimestamps() {return true;}
+
+  contacts() {
```

Comment:
This shouldn't be called contacts. A contact can't have it's own contacts. Also since you're referencing user, it's not even a contact, it's the user that this contact belongs to. What should this be called?